pkg/cli/offerings.go

```diff
@@ -0,0 +1,18 @@
+package cli
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func NewOfferings(c CommandContext) *cobra.Command {
```

Comment:
is there a need for autocompletion for offerings?

Reply:
Not right now since it just a housing for other subcommands (right now with my and @thedadams's PRs it will be WorkloadClass and VolumeClass, both of which do have autocompletion)